tests/test_refimpl_x_gts_ref.py

```
@@ -380,7 +380,472 @@ def test_start(self):
+        ),
+        Step(
+            RunRequest("validate anyOf instance - no branch matches should fail")
+            .post("/validate-instance")
+            .with_json({
+                "instance_id": "gts.x.testref_comb._.anyof_ref.v1~x.vendor._.i2.v1.0"
+            })
+            .validate()
+            .assert_equal("status_code", 200)
+            .assert_equal("body.ok", False)
+        ),
+    ]
```

Ruff (0.15.0)

[warning] Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)